src/components/Article.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import getArticle from "../assets/Utils/getArticle";
+
+const Article = () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+  useEffect(() => {
+    getArticle(article_id).then((article) => {
+      setArticle(article);
+    });
+  }, [article_id]);
```

Comment:
As you'd change pages to get a new article this doesn't need to be in the array. The array must exist but doesn't always need elements